hack/virtc/main.go

```diff
@@ -227,7 +228,14 @@ func buildPodConfig(context *cli.Context) (vc.PodConfig, error) {
 		Memory: vmMemory,
 	}
 
+	id := context.String("pod-id")
```

Comment:
@sameo - In terms of the command-line API then, are you saying that we should not have a `--id=` for `pod create` or `container create`? In other words, we should have the following behaviour:

- `virtc pod create`
  (generates pod `uuid` automatically).

- `virtc container create --pod-id=$pod_id`
  (generates container `uuid` automatically).